### ISSUE TITLE
Reports

### ISSUE THREAD

**Issue by DAPPERstats** [ddalthorp/GenEst#16]
One thing we have planned to include, but put on the back-burner due to Portugal was including a capacity to have a report tab that brings the results together and then allows the user to save a document. Can we come up with some design specs for this? What do we want to include?

**Comment by mhuso** [comment]
I'll take a first stab and we can discuss as a group.

**Comment by DAPPERstats** [comment]
bumping this
@mhuso do we want to include this in version 1 or not worry about it until version 2? can you mark it as the appropriate milestone?

**Comment by mhuso** [comment]
I think this is what Paul took a first stab at. I've made some comments in it.
[GenEstOutputExample.pbr.mmh.xlsx](https://github.com/ddalthorp/GenEst/files/1757223/GenEstOutputExample.pbr.mmh.xlsx)

**Comment by mhuso** [comment]
Arrgh! I thought "Close and Comment" simply means I was going to the next topic. I didn't mean to close it completely! Sorry.

**Comment by DAPPERstats** [comment]
yeah, that's the doc I was thinking about, my question though was "do we want that functionality in version 1 that we will be passing around for beta or do we want to hold off on that for now and include in the next version?" (basically asking about the priority here)

No worries!

**Comment by mhuso** [comment]
I'm fine with putting the Report Tab on the back-burner for now. The beta testers will be able to see what sorts of output tables we provide without necessarily needing to see the polished "pull them all together into a single report" form. They will probably ask for something like that, so we'll be poised to give it to them :-)
